internal/tar_ball_file_packer.go

```diff
@@ -141,10 +133,6 @@ func (p *TarBallFilePacker) createFileReadCloser(cfi *ComposeFileInfo) (io.ReadC
 			return nil, errors.Wrapf(err, "PackFileIntoTar: failed to find corresponding bitmap '%s'\n", cfi.path)
 		}
 		fileReadCloser, cfi.header.Size, err = ReadIncrementalFile(cfi.path, cfi.fileInfo.Size(), *p.incrementFromLsn, bitmap)
-		if os.IsNotExist(err) { // File was deleted before opening
```

Comment:
I believe the file can be deleted between delta construction and opening for a read. Both places are important.